### ISSUE TITLE
Spec: Fix the "Format of copyright notices" section

### ISSUE THREAD

**Issue by carmenbianca** [fsfe/reuse-docs#33]
Thread: https://github.com/fsfe/reuse-docs/pull/23#pullrequestreview-242358607

Relevant section:

```
The copyright notice MUST contain the year(s) of publication, the name of the
copyright holder, and the contact address of the copyright holder. The order of
these items MAY be changed, but SHOULD be consistent within the Project. The
items except the name of the copyright holder MAY be omitted.
```

Suggested fix:

```
The copyright notice SHOULD contain the year(s) of publication, the name of the
copyright holder(s), and the contact address of the copyright holder(s); whereby the name of the copyright holder(s) MUST be included. The order of
these items MAY be changed, but SHOULD be consistent within the Project.
```

@silverhook @carmenbianca

**Comment by carmenbianca** [comment]
I'm a little on the fence about this one. In essence I agree with:

- SHOULD for year and contact
- MUST for copyright holder

The rest, I feel, can be improved:

- The more I'm reading this, it appears to me that "SHOULD be consistent within the Project" is a fool's errand when you factor in third-party copyright notices. This should probably be removed.

- "The order of these items MAY be changed" --- I think it would be better if this were changed into a SHOULD. e.g., the items SHOULD be in such-and-so order. This is an improvement, because the current wording only implicitly recommends an order.

- Allowing a plural of "copyright holder(s)" makes the sentence a lot more tedious to read, because "name(s)" and "address(es)" should then also be plural. I think it would be okay to keep this singular. Alternatively, add an extra statement at the end saying that a copyright holder MAY also be joint copyright holders.

- Should the contact address be in between angle brackets?

So perhaps something like:

---

The copyright notice MUST contain the name of the copyright holder. The copyright notice SHOULD contain the year(s) of publication and the contact address of the copyright holder. The order of these items SHOULD be: year(s), name, contact address.

The contact address of the copyright holder SHOULD be in between angle brackets.

TODO: Figure out a way to phrase "copyright holder" MAY also be "joint copyright holders". English is hard.

**Comment by carmenbianca** [comment]
Figured out a wording.

---

A copyright notice MAY contain joint copyright holders.

---

That wasn't so hard.

**Comment by silverhook** [comment]
Careful, co-authorship and joint authorship is not the same. I would keep it simple and simply talk about “copyright holder(s)” and leave it it that. How exactly the rights and relationships between the individual (copy)right-holders in the project lies, is outside the scope of REUSE. All that matters is that they are easy to find.

How about:

```
The copyright notice MUST contain the name of the copyright holder(s). The copyright notice 
SHOULD contain the year(s) of publication and the contact address(es) of the copyright holder(s). 
The order of these items SHOULD be: year(s), name(s), contact address(es).

Any contact address SHOULD be in between angle brackets.
```

Or:


```
The copyright notice MUST contain the name of the copyright holder. The copyright notice 
SHOULD contain the year(s) of publication and the contact address of the copyright holder. 
The order of these items SHOULD be: year(s), name, contact address.

Any contact address SHOULD be in between angle brackets.

Copyright holder MAY be an individual, list of individuals, group, legal entity, or any other 
descriptor by which one can easily identify the copyright holder(s).
```

---

As a pet peeve, I find it odd that we care more about having more years listed than more copyright holders.

**Comment by carmenbianca** [comment]
> As a pet peeve, I find it odd that we care more about having more years listed than more copyright holders.

This prompted me to have another think. I'd half-argue that the phrasing "year(s)" is ambiguous for our aims. e.g., is the plural optional or mandatory. So building on your second proposal, maybe something like:

```
The copyright notice MUST contain the name of the copyright holder. The copyright notice 
SHOULD contain the year of publication and the contact address of the copyright holder. 
The order of these items SHOULD be: year, name, contact address.

The year of publication MAY be a single year, multiple years, or a span of years.

The copyright holder MAY be an individual, list of individuals, group, legal entity, or any other 
descriptor by which one can easily identify the copyright holder(s).

Any contact address SHOULD be in between angle brackets.
```

I don't want to mandate the syntax (e.g., comma-separated for multiple years, dashes for ranges), though. Well, I want to, but it'd be a fool's errand.

(Also re-ordered to be in the same order of preference)

**Comment by silverhook** [comment]
As @carmenbianca, I’m not 100% happy with your suggestion in the https://github.com/fsfe/reuse-docs/issues/33#issuecomment-500781818 , but am happy enough to agree it is a good and very much workable compromise.

**Comment by carmenbianca** [comment]
Closed by 1acaaed
